routes.php

```diff
@@ -84,6 +84,11 @@
 // Router::add('/documents', 'pages/shared_documents');
 // Router::add('/download_shared_files', 'uploads/shared_docs/download_shared_file');
 
+//Report
+Router::add('/feedbacks', 'pages/feedbacks');
+Router::add('/feedbacks_list', 'pages/feedbacks_list');
+Router::add('/feedbacks_list/$user_id', 'pages/feedbacks_list');
```

Comment:
Not super important since this is only an admin page, but consistency doesn't hurt :)
```suggestion
Router::add('/feedback-list', 'pages/feedback_list');
Router::add('/feedback-list/$user_id', 'pages/feedback_list');
```

Reply:
Done